source/about/rules.rst

```diff
@@ -46,10 +46,13 @@ list of do's and don'ts. This stuff should all be common sense.
   whatever. Some exceptions are made in *Off-Topic* sections and are solely at the discretion of moderation staff.
 
 * Do not attempt to make a sale or sell anything on Sponge websites or systems. This includes but is not limited to
-  plugins (mods or any variation of the term), art assets, services, or any work to be provided. If it comes to our
-  attention that a plugin hosted by Sponge is being used as a promotion as a for-sale "lite" version, we reserve the
-  right to remove the plugin listing. What constitutes as a for-sale "lite" version will be decided at the discretion
-  of Sponge Staff.
+  plugins (mods or any variation of the term), art assets, services, or any work to be provided. We specifically
+  *allow* offering a bounty in an *original post* in the **Devs Wanted** area, and developers may seek commissioned work
+  in the **Devs Available** category. Any further discussion of payment in these (or any other) threads is prohibited.
+
+* If it comes to our attention that a plugin hosted by Sponge is being used as a promotion as a for-sale "lite" version,
```

Comment:
Shouldn't it be more like "a for-sale "premium" version?

The "lite" version would be the one hosted by Sponge, with the "premium" available off platform.